### ISSUE TITLE
Create And connect an user

### ISSUE THREAD

**Issue by peegee1985** [maierfelix/POGOserver#305]
Hello, please Tell mě how the hack do i create user to play? Username And password is not aby of the values in the db. How to create player?

**Comment by DracoMilesX** [comment]
You need to use an already existing user by logging in with google to the official servers. Then disconnect and open up the xposed module and fill in where it needs to redirect too. Then start PoGO again and you should connect to your private server.

**Comment by peegee1985** [comment]
Hello, thanks for an answer. I should connect via google on server or on he
client (nox app)

Dne 14. 11. 2016 3:44 odpoledne napsal uživatel "DracoMilesX" <
notifications@github.com>:

**Comment by DracoMilesX** [comment]
You need to register  your google account with the emulator or device your using. Then login on the official servers. Then close PoGO and use xposed module to redirect and connect to private server.

**Comment by peegee1985** [comment]
Its nit working. Client (apk) sayz unabke to authenticate... Even tho its pointed to my virtuální wattos server...

**Comment by peegee1985** [comment]
Could it be my server version 0.6.0 ?

**Comment by peegee1985** [comment]
And my emulated ios on nic app is 4.4.2

**Comment by peegee1985** [comment]
Sorry for my bad  english. My android is 4.4.2 on nox app And xposed redirected to my server. Its saying unable to authenticate.

**Comment by peegee1985** [comment]
And client version is 0.35

**Comment by DracoMilesX** [comment]
Dunno what does it say on the server console? Otherwise it might be the server version haven't tested it in a while.

**Comment by peegee1985** [comment]
I think There is a problem, when i do npm run Bolt it should download all the db things from Niantic , right? But There is nothing in my sql db nore in my src folder. ...im confused .

**Comment by peegee1985** [comment]
Npm run boot

**Comment by peegee1985** [comment]
And im running this...https://github.com/maierfelix/POGOserver/wiki/Pre-Installed-POGOserver-on-a-Oracle-VM-image-with-WattOS-R10

**Comment by DracoMilesX** [comment]
It should if you have modified the config file. It should create a folder data or models and the output should be seen in the console.

Which ip are you using the one you see in the vm or something else? Haven't tried the VM yet will try it tomorrow otherwise maby @Badley187 can help u with it since he created it.

**Comment by peegee1985** [comment]
Should i mess with ip adress ať the nox app? Because There is ip way different from the one the Pogo server showing (listening at). Its connecting good to Niantic official (the nox app Pogo client) but since i change it to my Virtuální box server, it sayz unable to authenticate....

**Comment by DracoMilesX** [comment]
Yeah you can try it usually the ip it's listening on is not the correct one for virtual machines. You might as well need to put the network adapter on bridge.

**Comment by peegee1985** [comment]
OK i have server running but i have other issue. I have added spawnpoints. I see Pokémon in sightings. But on the game map its showing only those White circles And nothing is spawning. Why is that?

**Comment by chaze117** [comment]
The best way to connect VB server is using the Bridged adapter setting.
The white circles bug is appearing for me when I connect to the server from outside of my local nerwork.
